### PR TITLE
ocamlopt: add page

### PR DESCRIPTION
**Pull request by mira01**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/common/ocamlopt.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# ocamlopt
+
+> The OCaml native-code compiler.
+> Produces native executables out of OCaml files.
+
+- Create named executable from source file:
+
+`ocamlopt -o {{output_name}} {{source_file.ml}}`
```
Comment (sbrl) [inline pages/common/ocamlopt.md]: Could we tweak the values in the token syntax please? Just to make it more consistent with other pages.
```suggestion
`ocamlopt -o {{path/tooutput_file}} {{path/to/source_file.ml}}`
```

Also, are there any other examples we could include in this page? Seems a bit short with just the one.

Comment (pxgamer) [reply pages/common/ocamlopt.md]: Just updated your comment as it had a missing slash, hope you don't mind. 🙂

Comment (sbrl) [reply pages/common/ocamlopt.md]: Thanks, @pxgamer :heart:

**Comment by schneiderl** [comment]
Looks good to me but more examples would be good if that applies.

File: pages/common/ocamlopt.md (hunk 1 above)
Comment (agnivade) [reply pages/common/ocamlopt.md]: Let's name it to path/to/binary to be clear.

Comment (sbrl) [reply pages/common/ocamlopt.md]: Updated again

**Comment by sbrl** [comment]
That's a shame, I don't have permission to commit to this branch. Any reason you disabled that, @mira01? I was going to apply the suggestions and merge it, but as it stands that a bit of a challenge.

**Pull request closed without merging**